src/plugins/mdn/mdnPlugin.js

```diff
@@ -100,20 +35,20 @@ const mdnPlugin = async (msg) => {
     }
   }
 
-  if (res) {
-    res = await fixLanguage(res, lastRedirect).catch(() => null);
-  }
-
   if (!res || !res.ok) {
     msg.respondWithMention(`Try ${initialUrl} (couldn't fetch metadata)`);
     return;
   }
 
   let pageData;
   try {
-    pageData = extractFromHtml(res.text);
+    pageData = {
+      title: res.body.documents[0].title,
+      text:  res.body.documents[0].excerpt.replace(/<\/?mark>/g, ''),
```

Comment:
Should we be stripping all HTML here?

Reply:
Oh, you can pass `highlight=false` in the query to not get these `<mark>` elements.

https://developer.mozilla.org/en-US/docs/MDN/Contribute/Tools/Search

Reply:
good to know. just fixed it.